app/views/admin/_metadata_summary_card.html.erb

```diff
@@ -2,45 +2,52 @@
   summary_card_actions ||= []
 %>
 
+<% related_links_value = capture do %>
+  <% if schema["related"] %>
+    Yes
+    <% schema["related"].each_with_index do |content_id, index| %>
+      <p>Link <%= index + 1 %>: <%= content_id %></p>
+    <% end %>
+  <% else %>
+    No
+  <% end %>
+<% end %>
+
 <%= render "govuk_publishing_components/components/summary_card", {
   title: "Finder details",
   rows: [
-    {
+    *([{
       key: "Title of the finder",
       value: schema["name"]
-    },
-    {
+    }] if !(defined? previous_schema) || schema["name"] != previous_schema["name"]),
```

Comment:
The `*([{` syntax really threw me 😅 Trying this out locally, I think we can achieve the same outcome by dropping the `*` and the `[`...

```ruby
(
  {
    key: "Title of the finder",
    value: schema["name"],
  } if !(defined? previous_schema) || schema["name"] != previous_schema["name"]
),
```

Reply:
I've been trying locally and I don't think that you can construct an array with conditional elements like that.
But feel free to update the PR code if you have a working example.

Reply:
It seems to work for me locally, unless I'm not checking the right screens? (FWIW I'm trying this on the AAIB finder, and have tried editing the title and slug):

https://github.com/alphagov/specialist-publisher/pull/2829

Reply:
Ah, it was the `.compact` call that I was missing. Nice! That's much more readable.